Panic on `dstask show-next`

When executing `dstask show-next`, the following output appears:

```
panic: this should never happen?

goroutine 1 [running]:
main.main()
	./main.go:167 +0x1888
```

I suspect that this is due to the fact that `show-next` is an unhandled command.

--- a/cmd/dstask/main.go
+++ b/cmd/dstask/main.go
@@ -39,7 +39,7 @@ func main() {
 
 	switch query.Cmd {
 	// The default command
-	case "", dstask.CMD_NEXT:
+	case "", dstask.CMD_NEXT, dstask.CMD_SHOW_NEXT:
 		if err := dstask.CommandNext(conf, ctx, query); err != nil {
 			dstask.ExitFail(err.Error())
 		}

